### DISCUSSION TITLE
Cannot run simulations using sbert feature extraction.

### DISCUSSION THREAD

**Discussion by pablov-1995** [asreview/asreview#628] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
When I try to run a simulation passing sbert as a parameter for feature_extraction, I get the following error.

`KeyError: "None of ['dataset_name'] are in the columns"`

Do you know what could be happening?

**Comment by Rensvandeschoot** [comment, marked as answer]
Dear @pablov-1995  thank you for your issue. Probably you are missing a package in your environment necessary to run _sbert_, the name of the pakage is > sentence-transformers. See our documentation with the dependencies: https://asreview.readthedocs.io/en/latest/API/cli.html#feature-extraction

**Reply by pablov-1995** [reply]
It works now, thank you very much!

**Reply by J535D165** [reply]
@pablov-1995 can you create an issue in asreview about this uninformative error? Error should be self-explanatory, which it is not.
